tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+Suite Setup         Set Library Search Order  SeleniumLibrary
+Suite Teardown      End Web Test
+
+
+*** Test Cases ***
+Verify Any OpenShift User Can Spawn JupyterLab Notebooks
+    [Documentation]    Verify that any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-1164
+    Verify Groups ConfigMap Contains The Expected OCP Groups
+    Verify User Can Spawn A Notebook
+
+
+*** Keywords ***
```

Comment:
I would recommend to keep tests and keywords in separated files. Mixing tests with keywords make it more difficult to reuse the keywords and to maintain the code.

Reply:
The keywords `Verify User Can Spawn A Notebook `  `Verify Groups ConfigMap Contains The Expected OCP Groups`  are specific to test.

Reply:
`Verify RHODS Groups ConfigMap Contains The Expected OCP Groups` could be in ODS.robot, as there are already other keywords related to rhods groups

You could rename it to 
```
Verify Default Access Groups Settings
   [Documentation]    Verifies that ODS contains the expected default groups settings
```

Also, instead of calling it from test `Verify Any OpenShift User Can Spawn JupyterLab Notebooks`, I think you  could create a different test `Verify Default Access Groups Setting` (or similar name in case there are conflicts with the keyword)  with different PolarionID in 110_rbac.robot

Reply:
Updated.

Reply:
Or maybe just change the title to `Verify Default Access Groups Settings And JupyterLab Notebook Access`

Reply:
Updated.